rules_test.go

```diff
@@ -0,0 +1,92 @@
+package core
```

Comment:
We are going to import this package as `import "github.com/scrapinghub/adblockgoparser"`. The library is going to have a name "adblockgoparser" but package points to core. So, users won't be able to import it:

```console
$ cat q.go
package main

import "github.com/scrapinghub/adblockgoparser"

func main() {
	adblockgoparser.ParseRule("aaa")
}

$ go build
go: finding github.com/scrapinghub/adblockgoparser latest
go: downloading github.com/scrapinghub/adblockgoparser v0.0.0-20200207122218-57b4f215969c
go: extracting github.com/scrapinghub/adblockgoparser v0.0.0-20200207122218-57b4f215969c
build github.com/scrapinghub/tt: cannot load github.com/scrapinghub/adblockgoparser: module github.com/scrapinghub/adblockgoparser@latest found (v0.0.0-20200207122218-57b4f215969c), but does not contain package github.com/scrapinghub/adblockgoparser
```

Please rename `core` to `adblockgoparser`.